### PR TITLE
Update French translation

### PR DESCRIPTION
**Pull request by pnax**
## Purpose

This PR updates French translations.

## Context

n/a

## Changes

fr.po

## How to test this PR

Review translated strings.

### CODE REVIEW

File: share/fr.po [hunk 1]
```diff
@@ -72,6 +72,10 @@ msgstr ""
 "Un nom ou une adresse IP correspondant à un serveur de noms pour les tests "
 "sur des zones non déléguées. Peut être utilisée plusieurs fois."
 
+msgid "Name of a root hints file to override the defaults."
+msgstr "Chemin du fichier d'indices de la racine remplaçant le fichier par "
```
Comment (marc-vanderwal) [inline share/fr.po:76]: I think “root hints” is better translated as “indications de la racine”. See for example https://www.bortzmeyer.org/8499.html.

File: share/fr.po [hunk 2]
```diff
@@ -200,6 +204,13 @@ msgstr ""
 "--level doit avoir pour valeur CRITICAL, ERROR, WARNING, NOTICE, INFO, "
 "DEBUG, DEBUG2 ou DEBUG3.\n"
 
+msgid ""
+"Only one domain can be given for testing. Did you forget to prepend an "
+"option with '--<OPTION>'?\n"
+msgstr ""
+"Uniquement un domaine peut être testé. Auriez-vous oublié de préfixer une "
```
Comment (marc-vanderwal) [inline share/fr.po:211]: How about “Seul un domaine peut être soumis à un test”?

File: share/fr.po (hunk 1 above)
Comment (pnax) [reply share/fr.po]: Thanks for the suggestion. Fixed.

File: share/fr.po (hunk 2 above)
Comment (pnax) [reply share/fr.po]: Updated.

**Review by marc-vanderwal — APPROVED** [review]
Looks good to me.

**Pull request merged**